include/zephyr/bluetooth/bluetooth.h

```diff
@@ -3118,7 +3130,8 @@ int bt_le_per_adv_set_response_data(struct bt_le_per_adv_sync *per_adv_sync,
  * @details Valid Bluetooth LE identity addresses are either public address or random static
  * address.
  *
- * @param id   Local identity handle (typically @ref BT_ID_DEFAULT).
```

Comment:
Why was the reference to `BT_ID_DEFAULT` removed?

Reply:
It was an attempt to make it more clear that the functions could be used for other identities, but yeah it might be better to add the "typically @ref BT_ID_DEFAULT" as well?

Reply:
I think so to. A user that does not use multiple identities is unlikely to know what identity value to set otherwise. 

An alternative would be add a reference to a section documentation identities in Zephyr